fe/fe-core/src/main/java/org/apache/doris/nereids/types/DataType.java

```diff
@@ -350,7 +350,7 @@ public static DataType fromCatalogType(Type type) {
         } else if (type.isStructType()) {
             List<StructField> structFields = ((org.apache.doris.catalog.StructType) (type)).getFields().stream()
                     .map(cf -> new StructField(cf.getName(), fromCatalogType(cf.getType()),
-                            cf.getContainsNull(), cf.getComment()))
+                            cf.getContainsNull(), cf.getComment() == null ? "" : cf.getComment()))
```

Comment:
document shows that comment is optional. How about remove the null check in Strcut creator ?

Reply:
in Nereids, we should not make any attr be null, it is hard to process and debug